pootle/static/js/editor/app.js

```diff
@@ -2209,26 +2247,45 @@ PTL.editor = {
 
 
   /* Accepts a suggestion */
-  acceptSuggestion(suggId, { skipToNext = false } = {}) {
-    UnitAPI.acceptSuggestion(this.units.getCurrent().id, suggId)
+  handleAcceptSuggestion(
+    suggId, { requestData = {}, isSuggestionChanged = false } = {}
+  ) {
+    PTL.editor.acceptSuggestion(suggId, { requestData, isSuggestionChanged });
+  },
+
+  acceptSuggestion(
+    suggId, { requestData = {}, skipToNext = false, isSuggestionChanged = false } = {}
+  ) {
+    if (!isSuggestionChanged) {
```

Comment:
I put it separate because these methods are called form different context. It was done as you are asking before and I found it confusing.